sass/_main.scss

```diff
@@ -196,6 +196,11 @@ body.essay {
             color: black;
         }
     }
+
+    h2 {
+      margin-top: 4rem;
```

Comment:
Careful! This pushes the titles of articles waaaay down.

You want:

```
body.essay h2:not(:first-of-type) {
    margin-top: 4rem;
}
```

Reply:
I guess since it's nested here you just need the `h2` onward.

Reply:
Oh, sure. Why is the first header in an article an h2? Shouldn't it be an h1?

Reply:
I don't remember, really. Might've just been a bad habit (I remember traditionally I would never use `h1`s), but since they're flattened h1 should be fine, but that would mean going back to what you had here for `h2`s and changing the title to an `<h1>` [here](https://github.com/urbit/urbit.org/blob/master/templates/essays/post.html#L47).

Reply:
I just used your suggestion, seems fine. From a DOM-semantics standpoint, the first header should be an h1 tho, I think.

Reply:
Yeah, I agree — I was just dictating how to do the correct, semantic, avoiding increasing complexity solution lol, I can just do this later if you prefer